### DISCUSSION TITLE
How to translate docs?

### DISCUSSION THREAD

**Discussion by BanCrash** [moneymanagerex/moneymanagerex#5407] [Q&A] [answered]
Hi! Not so long time ago the docs were added on Crowdin for their translation (though they seemed to not get updated on the program, since I modified them but they weren't updated on the repository), but now they're gone on Crowdin, so how we should translate them?

I'm talking about the help pages of the program:

![imagen](https://user-images.githubusercontent.com/11770760/206469146-6eb9ec9b-efa7-4c62-95d3-e6f94ce41517.png)

**Comment by vomikan** [comment, marked as answer]
https://github.com/moneymanagerex/moneymanagerex/issues/5217

**Reply by BanCrash** [reply]
So we can't translate them any more?

I stopped translated them because even the files where I reached the 100% of translations never got to the repository. I thought that was an issue that needed to be fixed and with the fear of the progress to get lost I stopped until the situation was fixed.

**Reply by vomikan** [reply]
The files are saved. If the translation is incomplete in the program, create a case and send an archive with the translated files. Let's upload them.

**Reply by BanCrash** [reply]
> The files are saved. If the translation is incomplete in the program, create a case and send an archive with the translated files. Let's upload them.

Ok, I will check which files needs to be updated, update and improve them and then upload them here when I am finished. For example I remember that I reached the 100% on grm_es_es doc to see if that was the reason of them not getting updated to the repository, but still needs minor improvements, so I'll start with that.

**Reply by BanCrash** [reply]
A quick update, after starting I've found that since I don't know which files are up to date and which not, I will start with the english version and translate directly or reuse the previous strings if they fit.

This is going to be a slow job so don't expect it any time soon since I don't have much free time now. I will open a PR with all the updates when I'm finished.
